src/NzbDrone.Core/Indexers/FetchAndParseRssService.cs

```diff
@@ -42,6 +45,13 @@ public List<ReleaseInfo> Fetch()
 
             foreach (var indexer in indexers)
             {
+                var indexerStatus = _indexerStatusService.GetIndexerStatus(indexer.Definition.Id);
+                if (indexerStatus != null && indexerStatus.DisabledTill.HasValue && indexerStatus.DisabledTill.Value > DateTime.UtcNow)
```

Comment:
maybe this should be moved to `indexerFactory.RssEnabled()` with a boolean flag `bool includedBlocked`